sdcm/loader.py

```diff
@@ -79,14 +80,16 @@ def run(self):
 
 
 if __name__ == "__main__":
+    # pylint: disable=ungrouped-imports
     import tempfile
     import logging
-    import requests
     import unittest
 
+    import requests
+
+    from sdcm.prometheus import start_metrics_server, nemesis_metrics_obj
+
```

Comment:
move to unit tests, if not needed remove the code

Reply:
move to test_manual.py